src/beacon/docker-compose.yml

```
@@ -1,33 +1,60 @@
 
 networks:
   airqo-network:
-    driver: bridge
\ No newline at end of file
+    external: ${NETWORK_EXTERNAL:-true}
+    name: ${NETWORK_NAME:-airqo-network}
+
+# Named volumes for better performance
+volumes:
+  node_modules:
+    driver: local
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)